sklearn/model_selection/_validation.py

```diff
@@ -729,14 +729,18 @@ def cross_val_predict(estimator, X, y=None, *, groups=None, cv=None,
               as in '2*n_jobs'
 
     method : str, default='predict'
-        Invokes the passed method name of the passed estimator. For
-        method='predict_proba', the columns correspond to the classes
-        in sorted order.
+        The method to be invoked by `estimator`.
 
     Returns
     -------
     predictions : ndarray
-        This is the result of calling ``method``
+        This is the result of calling ``method``. Shape:
+
+            - When `method` is predict: (n_samples, 1)
+            - When `method` is one of {'predict_proba', 'predict_log_proba',
+              'decision_function'}: (n_samples, n_classes).
```

Comment:
Not as simple. In binary classification, we have `(n_samples,)` instead of `(n_samples, n_classes)` for the `decision_function` if I recall properly.

Reply:
Yes you're right